### ISSUE TITLE
Improve focus state for all form controls

### ISSUE THREAD

**Issue by sebnitu** [sebnitu/vrembem#661]
## Problem

There are some accessibility issues with the current focus styles for a few of the form control components. There should be a full audit and update to these styles to help improve usability. Components to check include:

- button
- checkbox
- icon-action
- input
- menu
- radio
- switch

**Resources**

- https://developer.mozilla.org/en-US/docs/Web/CSS/outline
- https://dockyard.com/blog/2020/04/28/designing-for-accessibility-focus-states
- https://www.nicchan.me/blog/tips-for-focus-styles/
- https://css-tricks.com/focusing-on-focus-styles/
